install.sh

```diff
@@ -79,9 +79,17 @@ set -euo pipefail
   fi
   echo "release=$release"
 
+  if [[ -n "${DIRENV_GITHUB_API_TOKEN:-}" ]]; then
+    # note: this doesn't actually print the token value.
+    echo "using DIRENV_GITHUB_API_TOKEN for GitHub API authentication"
+    authorization_args="-H 'Authorization: Bearer ${DIRENV_GITHUB_API_TOKEN}'"
```

Comment:
```suggestion
    authorization_args=(-H "Authorization: Bearer ${DIRENV_GITHUB_API_TOKEN}")
```
If you make it an array it makes the escaping a bit clearer below.